x-pack/plugins/lens/public/_mixins.scss

```diff
@@ -1,13 +0,0 @@
-// sass-lint:disable-block indentation, no-color-keywords
-
-// SASSTODO: Create this in EUI
-@mixin lnsOverflowShadowHorizontal {
```

Comment:
Mixin and variable files should still be their own file (even if they're used by only a single component **right now**). Then what you do is import them in the SASS files that need them instead of relying on a manifest.